tools/rebuildPtreeRst.py

```diff
@@ -0,0 +1,42 @@
+import os.path
```

Comment:
Script seems fine to me, just highlights that we really should try to migrate toward using sphinx-autogen/autosummary. My issue is it usually seems to require some templating/theming which isn't really a passion of mine.

Reply:
Yeah, this is more of a stand-in but certainly not the best long-term solution